pkg/qdr/qdr.go

```diff
@@ -836,6 +837,19 @@ func (a *BridgeConfig) Difference(b *BridgeConfig) *BridgeConfigDifference {
 		HttpConnectors: a.HttpConnectors.Difference(b.HttpConnectors),
 		HttpListeners:  a.HttpListeners.Difference(b.HttpListeners),
 	}
+
+	for _, httpConnector := range result.HttpConnectors.Added {
+		if len(httpConnector.SslProfile) > 0 {
+			result.SslProfiles = append(result.SslProfiles, httpConnector.SslProfile)
```

Comment:
Is it possible that more than one connector could ever share the same ssl profile? (If so, profiles are only really added if nothing else is using them).